tuist/tuist
# Add configuration option to control location of SPM clones with Xcode native integration

Resolves: no issue raised (happy to do so if needed). See the Slack discussion [here](https://tuistapp.slack.com/archives/CCXC295QS/p1683878879654209)

### Short description 📝

I'm basically re-opening this pull request: https://github.com/tuist/tuist/pull/3628
It was closed due to no activity. After asking on Slack, it turns out there's no reason not to implement the change. It seems like other people would use it on projects that don't use `Dependencies.swift`

### How to test the changes locally 🧐

- Create an app using Tuist
- Add one or more package dependencies **without** using Dependencies.swift (with the `packages` parameter in `Project`)
- Set a custom `clonedSourcePackagesDirPath` in the project `Config`
- Run `tuist generate` and observe that the packages are cloned in the custom folder path (`<custom-path>/<workspace-name>`)

### Contributor checklist ✅


### Reviewer checklist ✅
